reconf/test_reconf_base.py

```diff
@@ -63,33 +47,1676 @@ def test_stop(self):
         tempesta.config.set_defconfig(self.tempesta_orig["config"])
         self.start_tempesta()
 
-        os.system("sysctl -e -w net.tempesta.state=stop")
+        remote.tempesta.run_cmd("sysctl -e -w net.tempesta.state=stop")
         tempesta.run_start()
 
-    def test_reconf_gt_socks(self):
+    def test_reconf_busy_socks(self):
+        """The user is trying to add listen to a busy port by another service."""
         tempesta = self.get_tempesta()
+        port_checker = FreePortsChecker()
+
+        self.start_all_servers()
+
+        # Tempesta listen 443 port and Nginx listen 8000 port
         tempesta.config.set_defconfig(self.tempesta_orig["config"])
         self.start_tempesta()
 
-        tempesta.config.set_defconfig(self.tempesta_alt_gt_socks["config"])
-        tempesta.reload()
+        # Tempesta listen 443, 8000, 4433 port and Nginx listen 8000 port
+        tempesta.config.set_defconfig(self.tempesta_busy_socks["config"])
+        self.oops_ignore = ["ERROR"]
+        with self.assertRaises(CmdError):
+            tempesta.reload()
+
+        with self.assertRaises(CmdError):
```

Comment:
According to our discussion if Tempesta can't reload because some sockets from config are already in use, Tempesta stop listen all sockets. So port=443 is not busy after reload fails.